include/pistache/listener.h

```diff
@@ -87,6 +88,7 @@ class Listener {
     std::thread acceptThread;
 
     size_t workers_;
+    char   workersName_[16];
```

Comment:
Please conform to the existing code style.  Only one space between type ("char") and name ("workersName").